test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
most TS code I've seen, including ambient typings on DefinitelyTyped, uses the `b?: number` convention with no space before the `?`. so I'm inclined to disagree with this for the core TS rule. if you wish to enforce `b ?: number` in your codebase, then I think you are best served by a custom rule.